docs/final report/chapters/feature-description.tex

```diff
@@ -1,11 +1,57 @@
+\subsubsection{Comparison}
+To determine a relation between events, the lag sequential comparison is available. The lag sequential analysis will return a table chronologically sorted on the events.
+%lag sequential
+\subsubsection{Code}
+Codes can be added to rows when a value of that row satisfies a constraint. A single row can contain zero or more codes. When a row contains a code, it can be selected via a specific constraint that selects only that code. 
+\subsubsection{Chunking}
+Real chunks are not supported by the program, however the group by function is. A group by function can be applied on a column or on a constraint combined with a column. This creates a new table with the first column the value of the group, the following columns are created after the functions given by the user, applied on that column.
+\subsubsection{Connections}
+To connect two or more tables, a join function is available. Tables can be joined in multiple ways. The full join is available as well as a left, right and simple join.
+\subsection{Additional Analyses}
+Apart from the implemented C's, more functions can be used in the program.  
+\subsubsection{Union}
+Two tables can be unified with the union command. This creates a new table with only the rows that have the same values in both tables. Codes of the rows of the two tables are not taken into account when comparing, but when two rows are the same the codes of both rows are combined in the new table.
+\subsubsection{Difference}
+The opposite of the union is also possible, the difference operation. Two tables are subtracted and only unique rows are added to the resulting table.
 \section{Visualize}
-\section{Export}
\ No newline at end of file
+No analysis is finished without visualizing the data. A visualization shows the data in an easy and fast interpretable way. In the visualization tab of the program buttons can be pressed to show a pop-up where the data for that visualization can be specified. After a visualization is made, the image can be saved as PNG file.
+\subsection{Box-plot}
+A single box-plot can be created of one column with the FloatValue or IntValue type. This happens in the graph pop-up where first a table is selected and then the column can be selected. The support of creating multiple box-plots at once is not implemented but it is possible to save box-plots one by one.
+\subsection{Bar-chart}
+A bar-chart can also be made via the graph pop-up. The bar-chart needs 2 columns to be created. For the x-axis a column is needed with distinct values, this is for example the column of a table where is grouped by chunk. For the y-axis a column with NumberValues must be picked. 
```

Comment:
this is for example the column of a table where is grouped by chunk. zin is een beetje raar,